src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ViewCellRenderer.cs

```diff
@@ -244,6 +244,12 @@ public void Update(ViewCell cell)
 
 				var platformView = _viewCell.View.ToPlatform(Element.FindMauiContext());
 				_viewHandler = (IPlatformViewHandler)_viewCell.View.Handler;
+
+				if (platformView?.Parent is ViewGroup)
```

Comment:
If you're using `RemoveFromParent` I don't think you need the if statement here

Reply:
@PureWeen, I have modified the code based on your concerns, could you please validate once and let me know if there are any concerns.